src/kraken/std/python/tasks/info_task.py

```diff
@@ -0,0 +1,83 @@
+import re
+import subprocess
+from pathlib import Path
+from typing import List
+
+from kraken.core.api import Project, Property, Task, TaskStatus
+
+from ..buildsystem import PythonBuildSystem
+
+
+def get_python_path() -> str:
+    """Returns Python path"""
+    path = run_command(["which", "python"])
+    if "command not found" in path:  # Windows
+        path = run_command(["where", "python"])
+    return path
```

Comment:
The python path which is interesting is the one that the managed environment uses (for debugging / development purposes), it's typically just in a subdir of the virtual environment (like `$VIRTUAL_ENV/bin/python`)

I think calling `which python` offers no guarantees regarding what it returns (it could be the system python, or anything else...)

```suggestion
    """Returns the path to the Python interpreter of the Kraken-managed environment."""
    return self.get_virtual_env_path() / "bin" / "python"
```

(+ moving it inside `InfoTask`)